lib/Reader/readerir.cpp

```diff
@@ -5863,7 +5883,11 @@ void GenIR::AddPHIOperand(PHINode *PHI, Value *NewOperand,
     LLVMBuilder->restoreIP(SavedInsertPoint);
   }
 
-  PHI->addIncoming(NewOperand, NewBlock);
+  int BlockIndex = PHI->getBasicBlockIndex(NewBlock);
+  if (BlockIndex >= 0)
+    PHI->setIncomingValue(BlockIndex, NewOperand);
+  else
+    PHI->addIncoming(NewOperand, NewBlock);
```

Comment:
Do we expect the `else` clause here to ever get hit anymore? 

If we're adding all the phi operands when we first create the PHI then it seems like we should never add another one here.